tcms/kiwi_auth/tests/test_forms.py

```diff
@@ -0,0 +1,68 @@
+import importlib
+
+from django.conf import settings
+from django.test import TestCase, override_settings
+
+from tcms.kiwi_auth import forms
+
+
+class TestRecaptchaField(TestCase):
+    def setUp(self):
+        self.data = {
+            "username": "test_user",
+            "password1": "password",
+            "password2": "password",
+            "email": "new-tester@example.com",
+        }
+
+    @override_settings(INSTALLED_APPS=settings.INSTALLED_APPS + ["captcha"])
+    def test_captcha_required_when_enabled(self):
+        importlib.reload(forms)
+
+        RegistrationForm = forms.RegistrationForm
```

Comment:
this is not necessary, you can use `forms.RegistrationForm` everywhere, no ?

Reply:
Fix everywhere and don't mark comments as resolved b/c it makes them harder to see. I have to expand all of them to see what's been done.